common-content/en/module/html-css/what-is-css/index.md

```diff
@@ -54,18 +56,22 @@ You learn them by using them. You look it up here: https://developer.mozilla.org
 
 <iframe src="https://giphy.com/embed/zFSGphnnT9opa" width="480" height="480" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/zFSGphnnT9opa">via GIPHY</a></p>
 
-A cascade is a stream or sequence of a things where each thing affects the next. You might imagine pouring champagne into a tower of glasses. The champagne flows into the top container; it then cascades into the next container; etc. In CSS we’re actually pouring from many bottles into one tower.
+A cascade is a stream or sequence of a things where each thing affects the next. You might imagine pouring champagne into a tower of glasses. The champagne flows into the top container; it then cascades into the next container; etc. In CSS we're actually pouring from many bottles into one tower.
 
 So in CSS 'the cascade' means the order or sequence of the rules as they flow into the DOM tree according to CSS precedence.
 
+Several rules may apply to one element, and they are applied in a particular order, with later, more specific, or more important rules adding to or overriding ones that have already been applied.
+
 </details>
 
 <details>
 <summary>What does precedence mean?</summary>
 
-It means hierarchy. One hierarchy in CSS is inheritance: children inherit from parents. If a parent node has red text, all its children will inherit that red text color. But when another rule comes after, and overrules, it takes precedence. Everything in CSS is determined by order.
+It means the order rules are applied.
+
+One way rules are applied in CSS is inheritance: children inherit from parents. If a parent node has red text, all its children will inherit that red text color. But when another rule with higher precendence comes and overrules, it replaces the rule which had lower precedence. An explicit rule targeting an element directly overrules a rule it inherited from its parent.
```

Comment:
Hmm, why are we not telling the word hierarchy? It will come up a lot in CSS.

Reply:
I'm happy to re-add the word hierarchy, perhaps in another Q&A or a separate sentence?

But I found the existing phrasing quite hard to take meaning from when trying to understand the word precedence. In particular, I thought the existing answer was trying to smoosh together the concepts of precedence, hierarchy, inheritance, and order in a very short space of words, which left me with more questions than answers :)

Reply:
Added a couple of sentences introducing hierarchies - I'm going to merge this for now, and we can keep editing as we see fit moving forwards :)